java/clients/pom.xml

```diff
@@ -253,6 +253,12 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-shade-plugin</artifactId>
+                <configuration>
+                    <transformers>
+                        <transformer
+                            implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer"/>
+                    </transformers>
+                </configuration>
```

Comment:
Should this configuration be applied to all the fat jars? We could put it in the fat-jar execution configuration in the parent pom?

Reply:
I'd suggest deciding which shaded jars to apply it to on a case-by-case basis for now.